lib/Stow.pm.in

```diff
@@ -617,6 +625,7 @@ sub unstow_contents_orig {
         next NODE if $node eq '..';
         my $node_target = join_paths($target, $node);
         next NODE if $self->ignore($stow_path, $package, $node_target);
+        # $node_target = adjust_dotfile($node_target);
```

Comment:
We don't want this line, but it will be cancelled out by the next commit when you squash the two commits into one anyway, so that's OK.